Language/Functions/USB/Keyboard/keyboardModifiers.adoc

```diff
@@ -14,57 +14,82 @@ title: Keyboard Modifiers and Special Keys
 
 [float]
 === Description
-The `Keyboard.write()` and `Keyboard.press()` and `Keyboard.release()` commands don’t work with every possible ASCII character, only those that correspond to a key on the keyboard. For example, backspace works, but many of the other non-printable characters produce unpredictable results. For capital letters (and other keys), what’s sent is shift plus the character (i.e. the equivalent of pressing both of those keys on the keyboard).
+When given a printable ASCII character as an argument, the functions `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()` simulate actuations on the corresponding keys. These functions can also handle ASCII characters that require pressing a key in combination with Shift or, on international keyboards, AltGr. For example:
+[source,arduino]
+----
+Keyboard.write('a');  // press and release the 'A' key
+Keyboard.write('A');  // press Shift and 'A', then release both
+----
+A typical keyboard, however, has many keys that do not match a printable ASCII character. In order to simulate those keys, the library provides a set of macros that can be passed as arguments to `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()`. For example, the key combination Shift-F2 can be generated by:
```

Comment:
```suggestion
A typical keyboard, however, has many keys that do not match a printable ASCII character. In order to simulate those keys, the library provides a set of macros that can be passed as arguments to `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()`. For example, the key combination Shift+F2 can be generated by:
```
I think `+` is the common convention for keyboard shortcut combination indicator character. For example:
https://github.com/github/docs/blob/main/contributing/content-style-guide.md#usage-highlights